server/opts.go

```diff
@@ -2482,6 +2486,9 @@ func setBaselineOptions(opts *Options) {
 	if opts.ReconnectErrorReports == 0 {
 		opts.ReconnectErrorReports = DEFAULT_RECONNECT_ERROR_REPORTS
 	}
+	if opts.MaxTracedMsgLen == -1 {
+		opts.MaxTracedMsgLen = MAX_TRACED_MSG_LEN
```

Comment:
This feels too low for the default. The behavior now is to print the whole msg which I think we need to preserve.

Reply:
Agree. I think we should not set baseline for `MaxTracedMsgLen`.

I have update the help info for `max_traced_msg_len` to add `-1 for unlimited` for describing that `-1` is the default value for `max_traced_msg_len` and it defaults to unlimited. This should be ok for now.